sdks/python/setup.py

```diff
@@ -271,18 +271,13 @@ def get_portability_package_data():
   return files
 
 
-python_requires = '>=3.8'
+python_requires = '>=3.9'
```

Comment:
(drive by): line 276 should have `>= 13` or `> 12`

Reply:
fixed